arch/arc/core/vector_table.c

```diff
@@ -65,3 +65,31 @@ struct vector_table _VectorTable _GENERIC_SECTION(.exc_vector_table) = {
 	0
 };
 
+#ifdef CONFIG_ARC_NORMAL_FIRMWARE
```

Comment:
Assume this is for the temporary secure initialization code, it's not really 'logical' to declare secure stuff in a 'NORMAL_FIRMWARE' conditional section. (maybe add a 'TODO' note)